x-pack/plugins/secops/public/components/page/overview/overview_host_stats/index.test.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import * as React from 'react';
+import { Provider as ReduxStoreProvider } from 'react-redux';
+
+import { mockGlobalState } from '../../../../mock';
+import { createStore, State } from '../../../../store';
+
+import { SiemOverviewHostStats } from '.';
+import { mockData } from './mock';
+
+describe('Overview Host Data', () => {
+  const state: State = mockGlobalState;
+
+  let store = createStore(state);
+
+  beforeEach(() => {
+    store = createStore(state);
+  });
+
+  describe('rendering', () => {
+    test('it renders the default SiemOverviewHostStats', () => {
+      const wrapper = shallow(
+        <ReduxStoreProvider store={store}>
```

Comment:
Not necessary here, but just a heads up in case you haven't come across it yet, @andrew-goldstein recently made a [TestProviders](https://github.com/elastic/kibana/blob/feature-secops/x-pack/plugins/secops/public/mock/test_providers.tsx) helper HoC for encapsulating some of the more common providers used in tests like this `ReduxStoreProvider`. It also has the `ThemeProvider`, `KibanaContextProvider`, and `DragDropCongtext` as well, so quite useful for keeping tests clean when multiple providers are required.